### DISCUSSION TITLE
On backend license

### DISCUSSION THREAD

**Discussion by boomzero** [XMOJ-Script-dev/XMOJ-Script#288] [General]
@langningchen Do you agree to relicense the backend to [AGPL](https://github.com/XMOJ-Script-dev/XMOJ-bbs/blob/master/LICENSE)

**Comment by langningchen** [comment]
Yes, I totally agree. @boomzero

**Comment by langningchen** [comment]
However, it's your decision. Do whatever you want except making the script worse.

**Comment by langningchen** [comment]
I'm sorry for being late to reply.

**Comment by boomzero** [comment]
However, I need your permission to relicense your commits

**Comment by langningchen** [comment]
If we follow the license, it looks like this. But sometimes, doesn't it have to be so strict. 🤣

**Comment by boomzero** [comment]
Legally

**Comment by boomzero** [comment]
(

**Comment by boomzero** [comment]
Also can you star this repo? Thx @langningchen

**Comment by langningchen** [comment]
Done

**Comment by boomzero** [comment]
How did you download Github mobile? When I try it shows “This app isn’t available in your country or region”...

**Reply by langningchen** [reply]
@boomzero 下载很简单，找个apk直接下就好了，安装使用的时候应该不会出问题

**Comment by boomzero** [comment]
Emm, iOS

**Reply by langningchen** [reply]
@boomzero 那我就没办法了，我家没有苹果设备

**Comment by boomzero** [comment]
。
